data/user_data_fetch_github.ps2

```diff
@@ -0,0 +1,17 @@
+<powershell>
+# Fetch Windows User Data from GitHub
+
+$HUDL_DIRECTORY = "c:\\hudl"
+$USER_DATA_URI = "https://s3.amazonaws.com/hudl-chef-artifacts/windows/user_data_base.ps2"
+$USER_DATA_FILE_PATH = "c:\\hudl\\user_data_base.ps1"
+
+if(!(Test-Path -Path $HUDL_DIRECTORY )){
+    New-Item -ItemType directory -Path $HUDL_DIRECTORY
+}
+
+Write-Output "Fetching user data from GitHub"
+Invoke-WebRequest $USER_DATA_URI -OutFile $USER_DATA_FILE_PATH
+
+Write-Output "Executing user data"
+Invoke-Expression $USER_DATA_FILE_PATH
```

Comment:
Could you check if t he results of running the user_data_base.ps1 get logged to the same log file? I'm guessing not, so we should log it somewhere.